unitTests/ui/ComponentEventsTest.ts

```diff
@@ -28,6 +29,36 @@ export function ComponentEventsTest() {
       expect(spy).toHaveBeenCalledWith({ bar: 'baz' });
     });
 
+    it('should execute handler with the originalEvent if its wrapped by jQuery and does not have any param', function() {
```

Comment:
Hey @louis-bompart, the logic looks good but it would be much cleaner if you used a `describe` block with a `beforeEach` and `afterEach` for the setup and teardown.

Inside it you would then have three `it` blocks. The comments would become test descriptions. e.g.

```
it(`when triggering a native click, it calls the spy with native event as the only parameter`)
it(`when triggering a JQuery click event without params, it calls the spy without any params`)
it(`when triggering a JQuery click event with a param object, it calls the spy passing the param object`)
```

Reply:
I really hesitated and based myself on the other tests, but wildo!